aii-core/src/main/eg/aii-shellfe.conf

```diff
@@ -15,3 +15,22 @@
 # The default is profile_ for backward compatibility
 # Comment this out when not prefix is needed
 profile_prefix = profile_
+
+# Directory on TFTP server where PXELINUX configuration files are.
+# Must be an absolute path or none to disable PXELINUX configuration.
+# nbpdir = /osinstall/nbp/pxelinux.cfg
+
+# Directory on TFTP server where Grub2 PXE configuration files are.
+# Must be an absolute path or none to disable Grub2 PXE configuration.
+# Default value is based on 'pxelinux' option if defined with last
+# path token replaced by 'grub-efi' or '/osinstall/nbp/grub-efi' otherwhise.
+# nbpdir_grub2 =
+
+
+# Parent directory for the installation kernel/initrd to use with Grub2 EFI.
+# Must be an absolute path relative to TFTP server root.
```

Comment:
tftpd interprets all the paths to download as relative to its root (it may even be chrooted). I am opened to a better wording... but the fact is that it must be an absolute path (start with a /) interpreted "inside the tftp root"... For me relative to the TFTP server root was quite clear...